composer.json

```
@@ -0,0 +1,34 @@
+{
+    "name": "govcms/govcms-ci",
+    "type": "project",
+    "repositories": [
+        {
+            "type": "vcs",
+            "url": "git@github.com:govCMS/scaffold-tooling.git"
+        },
+        {
+            "type": "composer",
+            "url": "https://packages.drupal.org/8"
+        },
+        {
+            "type": "composer",
+            "url": "https://asset-packagist.org"
+        }
+    ],
+    "minimum-stability": "alpha",
+    "require": {
+        "govcms/scaffold-tooling": "dev-feature/validate-scripts"
```

This has been merged, better than we get a `scaffold-tooling` release out and pin to a proper release

---

New release is out for `scaffold-tooling`, let's update this.

```suggestion
        "govcms/scaffold-tooling": "~2"
```